recipes/task-execution/src/main/java/org/apache/helix/taskexecution/TaskExecutionDemo.java

```diff
@@ -121,7 +121,7 @@ private static void populateDummyData(TaskResultStore taskResultStore) throws Ex
     float fraudProbability = 0.01f;
     float clickProbability = 0.01f;
     int numImps = NUM_IMP_EVENTS;
-    Random rand = new Random();
+    SecureRandom rand = new SecureRandom();
```

Comment:
I wonder if SecureRandom is a must here. Are the random numbers here for security or cryptographic purpose?